client/my-sites/marketplace/pages/marketplace-product-details/index.tsx

```diff
@@ -43,16 +43,20 @@ import { isAtomicSiteWithoutBusinessPlan } from 'calypso/blocks/eligibility-warn
 import { requestEligibility } from 'calypso/state/automated-transfer/actions';
 import Notice from 'calypso/components/notice';
 import { eligibilityHolds } from 'calypso/state/automated-transfer/constants';
+import { getDefaultPluginInProduct } from 'calypso/my-sites/marketplace/marketplace-product-definitions';
 
 interface MarketplacePluginDetailsInterface {
-	marketplacePluginSlug: keyof PluginProductMappingInterface;
+	productGroupSlug: keyof IProductGroupCollection;
+	productSlug: keyof IProductCollection;
 }
 
 function MarketplacePluginDetails( {
-	marketplacePluginSlug,
+	productGroupSlug,
 }: MarketplacePluginDetailsInterface ): JSX.Element {
+	// TODO : Depend on product slug to show product details in the page
+	const productSlug = 'yoast_premium';
```

Comment:
Should this use the `YOAST_PREMIUM` constant?

Reply:
This is hardcoded for now and will be cleaned up in the next PR. This was to prevent the PR size from creeping.